scrape/scrape.go

```diff
@@ -593,6 +594,7 @@ type loop interface {
 	run(interval, timeout time.Duration, errc chan<- error)
 	stop()
 	getCache() *scrapeCache
+	disableStalenessMarkers()
```

Comment:
This name is a misleading, as it's only the staleness markers after the scrape stops that it is affecting

Reply:
update ; it is longer but explicit